contracts/staking/contracts/src/vaults/StakingPoolRewardVault.sol

```diff
@@ -91,14 +92,14 @@ contract StakingPoolRewardVault is
         onlyStakingContract
         returns (
             uint256 operatorPortion,
-            uint256 poolPortion
+            uint256 membersPortion
         )
     {
         // update balance of pool
-        Balance memory balance = balanceByPoolId[poolId];
-        (operatorPortion, poolPortion) = _incrementBalanceStruct(balance, amount, operatorOnly);
-        balanceByPoolId[poolId] = balance;
-        return (operatorPortion, poolPortion);
+        Pool memory pool = poolById[poolId];
+        (operatorPortion, membersPortion) = _incrementPoolBalances(pool, amount, operatorOnly);
```

Comment:
Non-blocking: Thoughts on passing a storage pointer to `_incrementPoolBalances` instead of a memory pointer? If all we do is pass the memory pointer to `_incrementPoolBalances` and then update afterwards then I think that this makes sense.

As the tag indicates, this is not a merge blocking comment, but I do think that it could be an improvement. Guess I missed this last review

Reply:
You can just pass in `poolById[poolId]`.